Used just for (deprecated) cygwin_conv_to_full_win32_path(); there's really
no need for that, as we use *NIX paths

--- origsrc/cgdb-0.6.5/various/util/src/fs_util.c	2009-10-05 05:23:32.000000000 -0500
+++ src/cgdb-0.6.5/various/util/src/fs_util.c	2010-01-17 21:53:21.778317500 -0600
@@ -33,6 +33,8 @@
 #include "fs_util.h"
 #include "logger.h"
 
+#undef HAVE_CYGWIN
+
 #define MAXLINE 4096
 
 int fs_util_is_valid ( const char *dir ) {
